pages/categorization.vue

```diff
@@ -0,0 +1,521 @@
+					v-on:paint-action="$store.dispatch('saveCurrentPaintInfo', $event)"
+					>
+				</coloring-listgroup>
+			</b-col>
+
+			<b-col cols="8">
+				<filedata-table
+					:fields="columnMatchTableFields"
+					:paintClass="getPaintClass"
+					:currentPalette="$store.state.pageData.categorization.current"
+					:defaultPalette="$store.state.pageData.categorization.default"
+					:tableData="tableDataFromTsvAndOrJson"
+					v-on:column-name-selected="tableClick($event)">
+				</filedata-table>
+			</b-col>
+
+		</b-row>
+
+		<b-row>
+			
+			<b-col cols="9"></b-col>
+			
+			<b-col cols="3">
+				<!-- Only enabled when at least one column has been categorized -->
+				<b-button
+					class="float-right"
+					:disabled="!readyForNextStepFlag"
+					:to="'/' + pageNames.annotation.location"
+					:variant="nextPageButtonColor">
+					Next step: Annotate columns
+				</b-button>
+			</b-col>
+			
+		</b-row>		
+
+	</b-container>
+
+</template>
+
+<script>
+
+	// String literals
+
+	export default {
+
+		name: "CategorizationPage",
+
+		created() {
+
+			// Determine page state from data contents and change to that new state
+			this.changeToNewState();
+		},		
+
+		data() {
+
+			return {
+
+				// Columns for file data table	
+				columnMatchTableFields: [
+
+					{ key: "column" },
+					{ key: "description" }
+				],
+
+				// Current state of the page
+				currentState: 0,
+
+				// Default table background color
+				defaultBackgroundColor: this.$store.getters.pageData.categorization.default.bColor,
+
+				// Full text name of this page
+				fullName: this.$store.getters.pageNames.categorization.fullName, 
+
+				// Status of the navbar item links for other pages
+				navItemsState: [
+					
+					{
+						enabled: true,
+						pageInfo: this.$store.state.pageNames.home
+					},
+					{
+						enabled: false,
+						pageInfo: this.$store.state.pageNames.annotation
+					},
+					{
+						enabled: false,
+						pageInfo: this.$store.state.pageNames.download
+					}
+				],
+
+				// Bootstrap variant color of the button leading to the categorization page
+				nextPageButtonColor: "secondary",
+
+				// Possible states of this page
+				possibleStates: {
+
+					STATE_NOCATEGORIES_PAINTED: 0,
+					STATE_ATLEASTONE_CATEGORY_PAINTED: 1 << 1
+				},								
+				
+				// Local reference to the page names in the store
+				pageNames: this.$store.getters.pageNames,
+
+				paintClasses: {
+
+					paint0: "column-paint-0",
+					paint1: "column-paint-1",
+					paint2: "column-paint-2",
+					paint3: "column-paint-3",
+					paint4: "column-paint-4",
+					paintDefault: "column-paint-default"
+				},
+
+				// Whether or not page has enabled access to the annotation page
+				readyForNextStepFlag: false,
+
+				// Data for the coloring listgroup
+				recommendedColumns: {
+					
+					names: [
+
+						"Subject ID",
+						"Age",
+						"Sex",
+						"Diagnosis",
+						"Assessment Tool"
+					],
+					backgroundColors: [
```

Comment:
If we wanted to use the same palette (for instance, on the annotation page) it could make sense to store the palette globally, but since the indices here align with the `recommendedColumns` (changing the name to `recommendedCategories`) there is no reason at the moment to do so.